docs/Developers/Debug_Bears.rst

```diff
@@ -122,3 +122,74 @@ Or to debug all bears specified by ``bears`` setting:
     bears = PEP8Bear,MypyBear
     files = <filename>
     debug_bears = True
+
+Inspect Bear settings
+---------------------
+
+External pdb command ``settings`` has been included to inspect the bear
+settings in the debugging environment.(``settings`` command display's the
+updated setting of a Bear)
+
+Bellow is the code of simple CharCheckBear which check whether the
+passed message contains character more than 79.
+
+.. code:: python
+
+    import logging
+
+    from coalib.bears.LocalBear import LocalBear
+
+
+    class CharCheckBear(LocalBear):
+        def run(self, filename, file, message: str, line_length: int = 79):
+            """
+            Detect if a passed string contain characters more than 79.
+
+            :param message:
+                enter the message.
+            """
+            if len(message)<=79:
+                yield self.new_result(message=message, file=filename)
+            else:
+                yield self.new_result(message='message contains character '
+                                              'more than 79', file=filename)
```

Comment:
1. `line_length` is not used
2. This example doesn't round up so nicely, `SpaceConsistencyBear` is similar, and what's especially weird is that you lint on `message`, not on `file` ;) Please provide a better example :+1: